### ISSUE TITLE
Intermittent CI failure - R Package Install in molecular-subtyping-ATRT

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/OpenPBTA-analysis#577]
The `Molecular Subtyping and Plotting - ATRT` step in CI is failing with:

```
Installing package into ‘/usr/local/lib/R/site-library’
(as ‘lib’ is unspecified)
Error in contrib.url(repos, type) : 
  trying to use CRAN without setting a mirror
Calls: install.packages -> startsWith -> contrib.url
Execution halted

Exited with code exit status 1
```

When running [`analyses/molecular-subtyping-ATRT/02-ATRT-molecular-subtyping-plotting.R`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/286ff25022930024bb9812e3cfad5410a2cf49c8/analyses/molecular-subtyping-ATRT/02-ATRT-molecular-subtyping-plotting.R), the first steps of which look like:

```R
# Install and load in ComplexHeatMap
if (!("ComplexHeatmap" %in% installed.packages())) {
  install.packages("ComplexHeatmap")
}
library(ComplexHeatmap)

if (!("matrixStats" %in% installed.packages())) {
  install.packages("matrixStats")
}

if (!("ggfortify" %in% installed.packages())) {
  install.packages("ggfortify")
}

library(ggfortify)
library(dplyr)
```

But simply rerunning the workflow works sometimes – may have to do with the Docker layer caching. It's also worth noting that all these packages should be installed because taking out the logic for the R package installation (https://github.com/AlexsLemonade/OpenPBTA-analysis/commit/41d380cd85865abc5afb4bf5eac05cadcd38d1c5) doesn't cause a failure: https://circleci.com/gh/AlexsLemonade/OpenPBTA-analysis/3445

**Comment by jaclyn-taroni** [comment]
Except sometimes the packages _are not_ installed:

```
Error in library(ComplexHeatmap) : 
  there is no package called ‘ComplexHeatmap’
Execution halted
```

https://circleci.com/gh/AlexsLemonade/OpenPBTA-analysis/3453

**Comment by jaclyn-taroni** [comment]
`ComplexHeatmap` was not installed in the `Dockerfile` and now telomerase activity prediction steps are failing with:
```
Error in library(circlize) : there is no package called ‘circlize’
Execution halted
```
Wondering if these two packages used to be dependencies of dependencies of packages we installed from GitHub tied to a specific commit but we don't pin things "all the way down" or something like that.

**Comment by jaclyn-taroni** [comment]
Closed via #580
